site/content/3.13/graphs/graph-analytics.md

```diff
@@ -59,9 +82,30 @@ Single server deployments using ArangoDB version 3.11 are not supported.
    engine and can then run various algorithms with different settings.
 5. Write the computation results back to ArangoDB.
 6. Delete the engine once you are done.
+{{< /tab >}}
+
+{{< /tabs >}}
 
 ## Authentication
 
+{{< tabs "platforms" >}}
+
+{{< tab "ArangoDB Platform" >}}
+You can use any of the supported authentication methods the ArangoDB Platform
```

Comment:
Let's use "available authentication methods" or something similar to avoid having "support" twice in the same sentence.